plugins/outputs/kafka/kafka.go

```diff
@@ -292,6 +301,16 @@ func (k *Kafka) Init() error {
 		k.TLSKey = k.Key
 	}
 
+	if k.Socks5ProxyEnabled {
```

Comment:
How does this get set by an end-user?

Reply:
That can be configured by writing `socks5_enabled: true` in the plugin configuration

```
type Socks5ProxyConfig struct {
	Socks5ProxyEnabled  bool   `toml:"socks5_enabled"`
	Socks5ProxyAddress  string `toml:"socks5_address"`
	Socks5ProxyUsername string `toml:"socks5_username"`
	Socks5ProxyPassword string `toml:"socks5_password"`
}
```

Reply:
Ah right, I see it in the sample config above, can you please add it to the README as well?

Reply:
Ah, forgot that! Added it just now.

Reply:
thank you!